src/htcondor_es/convert_to_json.py

```diff
@@ -798,26 +819,35 @@ def convert_to_json(ad, cms=True, return_dict=False, reduce_data=False):
     if ('Chirp_WMCore_cmsRun_ExitCode' in result) and (result.get('ExitCode', 0) == 0):
         result['ExitCode'] = result['Chirp_WMCore_cmsRun_ExitCode']
 
+    # Parse CRAB3 information on CMSSW version
+    result['CMSSWVersion'] = 'Unknown'
+    result['CMSSWSubVersion'] = 'Unknown'
```

Comment:
I don't understand the motivation behind the Sub and SubSub version?

Reply:
I'll rename them to `CMSSWMajorVersion`, and `CMSSWReleaseSeries`